Signal/src/environment/PropertyListPreferences.m

```diff
@@ -130,9 +120,21 @@ - (void)setHasRegisteredVOIPPush:(BOOL)enabled
     [self setValueForKey:PropertyListPreferencesKeyHasRegisteredVoipPush toValue:@(enabled)];
 }
 
+- (BOOL)loggingIsEnabled
+{
+    NSNumber *preference = [NSUserDefaults.standardUserDefaults objectForKey:PropertyListPreferencesKeyEnableDebugLog];
```

Comment:
> Hmm, so the migration happens after the "is logging enabled" preference is checked at app startup?

The migration needs to read the database, but the purpose of these changes are to enable logging before the database is set up. So it's a pickle. I think one-time logging generic startup information is a reasonable concession. Here's the output:

```
2017/01/23 20:07:19:147  [VersionMigrations] Checking migrations. currentVersion: 2.6.13.1, lastRanVersion: 2.6.12.0
2017/01/23 20:07:19:151  [OWSDatabaseMigrationRunner] Skipping previously run migration: <OWS100RemoveTSRecipientsMigration: 0x16e3ae60> {
    uniqueId = 100;
}
2017/01/23 20:07:19:152  [OWSDatabaseMigrationRunner] Skipping previously run migration: <OWS101ExistingUsersBlockOnIdentityChange: 0x16e0e2c0> {
    uniqueId = 101;
}
2017/01/23 20:07:19:153  [OWSDatabaseMigrationRunner] Running migration: <OWS102MoveLoggingPreferenceToUserDefaults: 0x16e43b50> {
    uniqueId = 102;
}
2017/01/23 20:07:19:202  [AppDelegate] applicationDidBecomeActive.
2017/01/23 20:07:19:360  [OWS102MoveLoggingPreferenceToUserDefaultsMigrationId] copying existing logging preference to NSUserDefaults
2017/01/23 20:07:48:517  [OWS102MoveLoggingPreferenceToUserDefaults] assigning existing value: 0
2017/01/23 20:08:02:004  [OWS102MoveLoggingPreferenceToUserDefaults] Disabling file logger after one-time log settings migration.
```